Tribler/Core/Modules/gigachannel_manager.py

```diff
@@ -39,8 +39,8 @@ def start(self):
                     not self.session.has_download(str(my_channel.infohash)):
                 torrent_path = os.path.join(self.session.lm.mds.channels_dir, my_channel.dir_name + ".torrent")
                 self.updated_my_channel(TorrentDef.load(torrent_path))
-        except:
-            pass
+        except Exception:
```

Comment:
Do we have a good reason to use generic exceptions here? I would prefer to catch specific exceptions.

Reply:
This is a critical GigaChannel Manager loop. If it fails once, the loop stops completely. Therefore, the broad exceptions. We want this to either run under any circumstances or kill Tribler completely.